Makefile

```diff
@@ -123,9 +123,7 @@ schema:	## Generate OpenAPI schema file
 	python scripts/generate_openapi_schema.py docs/openapi.json
 
 requirements.txt:	pyproject.toml pdm.lock ## Generate requirements.txt file containing hashes for all non-devel packages
-	pip-compile --generate-hashes pyproject.toml
-	# please note that it is also possible to use the following command, but it generates sligthly less readable output file:
-	# pdm export --prod --format requirements --no-extras --output requirements.txt
+	pdm export --prod --format requirements --output requirements.txt --no-extra
```

Comment:
should be `--no-extras` as `pdm export --help` says

currently it seems, that `--no-extra` works, but it's probably PDM bug that might be fixed later :)